sktime/utils/validation/series.py

```diff
@@ -27,6 +27,11 @@
 )
 
 
+def is_in_valid_index_types(x) -> bool:
+    """Check that the input type belongs to the valid index types."""
+    return isinstance(x, VALID_INDEX_TYPES)
```

Comment:
I would actually change this to `isinstance(x, VALID_INDEX_TYPES) or is_integer_index(x)` and then use it everywhere

Reply:
done